src/sidebar/components/annotation-viewer-content.js

```diff
@@ -26,17 +26,16 @@ function fetchThread(api, id) {
 
 // @ngInject
 function AnnotationViewerContentController(
-  $routeParams,
   store,
   api,
   rootThread,
   streamer,
   streamFilter,
   annotationMapper
 ) {
-  store.setAppIsSidebar(false);
+  store.clearAnnotations();
```

Comment:
This ensures that annotations are cleared if switching from the stream back to the annotations view via the browser "Back" button.